scripts/ci/qemu-run.sh

```diff
@@ -176,37 +173,36 @@ case "${ACTION}" in
 		;;
   os)
     MEMORY="4G"
-    QEMU_PARAMS="${QEMU_PARAMS_BASE} ${QEMU_PARAMS_OS}"
-    check_disks
+    QEMU_PARAMS="${QEMU_PARAMS} ${QEMU_PARAMS_OS}"
+    check_disks ${ACTION}
     ;;
   os_install)
     MEMORY="4G"
-    QEMU_PARAMS="${QEMU_PARAMS_BASE} ${QEMU_PARAMS_OS} ${QEMU_PARAMS_INSTALLER}"
-    check_disks
+    QEMU_PARAMS="${QEMU_PARAMS} ${QEMU_PARAMS_OS} ${QEMU_PARAMS_INSTALLER}"
+    check_disks ${ACTION}
     ;;
   *)
     echo "Action: ${ACTION} not supported"
     exit 1
 		;;
 esac
 
-# Check for the existence of OVMF_CODE.fd and OVMF_VARS.fs files
-if [ ! -f "OVMF_CODE.fd" ] || [ ! -f "OVMF_VARS.fd" ]; then
-    echo "The required files OVMF_CODE.fd and OVMF_VARS.fs are missing."
-    echo "Downloading files from the server..."
-    wget -O ./OVMF_CODE.fd https://github.com/Dasharo/edk2/releases/latest/download/OVMF_CODE_RELEASE.fd
-    wget -O ./OVMF_VARS.fd https://github.com/Dasharo/edk2/releases/latest/download/OVMF_VARS_RELEASE.fd
+# Check for the existence of coreboot.rom file
+if [ ! -f "coreboot.rom" ]; then
+    echo "The required file coreboot.rom is missing."
+    echo "Downloading from the server..."
+    wget -O ./coreboot.rom https://github.com/Dasharo/edk2/releases/latest/download/coreboot.rom
```

Comment:
FIXME: this will be on coreboot repo, not edk2!

Reply:
> It should, although I haven't tested it, I always had coreboot.rom in place when running qemu-run.sh.

Then maybe drop some link with "RC" build and we will find out?

Reply:
Added here: 694812854b83670eede86338eb8822bb76f550ec

Let's see

Reply:
GH runner fails with

```
 Could not access KVM kernel module: Permission denied
qemu-system-x86_64: failed to initialize kvm: Permission denied
```

I wonder why, the kvm switch was also there before

Reply:
But just a few minutes ago it used to start OVMF binaries just fine: https://github.com/Dasharo/open-source-firmware-validation/actions/runs/9618984974/job/26534054299#step:4:126

Ideas?

Reply:
Interesting. Installation of KVM probably requires a restart, and because this PR changed the way CI runs it is possible that it uses a new container, while previous ones reused existing one. I've tried restarting the CI to check if next run will pass, but it still fails.

I've moved KVM parameter from `QEMU_PARAMS_OS` to `QEMU_PARAMS_BASE`. I've also fixed an issue that dropped display mode [here](https://github.com/Dasharo/open-source-firmware-validation/pull/318/files#diff-69e6355529b2609a8ddcccd04f8cd299d0fe65c19c246c853a31b9eb96fe6818R178), this is the only difference I can think of. Not sure if `-nographic` may cause KVM permission error :man_shrugging:

Reply:
Oh, wait, CI starts it as `./scripts/ci/qemu-run.sh nographic firmware &`, so it didn't use KVM earlier. This can mess up SMM operations
...

Reply:
@krystian-hebel I let myself to rebase, I suspect that is why the 2nd run fails.

Reply:
Other than changing this link to non-rc release I think it is ready.